cpp/cpp_odbc/Library/cpp_odbc/multi_value_buffer.h

```diff
@@ -1,17 +1,18 @@
 #pragma once
 
 #include <vector>
+#include <cstdint>
 
 namespace cpp_odbc {
 
 struct writable_buffer_element {
 	char * data_pointer;
-	long & indicator;
+	intptr_t & indicator;
```

Comment:
I guess this should be `int64_t`, as the indicator is always an `SQLLEN` value and is not associated to any pointer type at all.